docs/knowledgebase/learn-substrate/off-chain-workers.md

```diff
@@ -46,3 +46,23 @@ should be implemented to determine what information gets into the chain.
 
 For more information on how to use off-chain workers in your next runtime development project,
 please refer to our [Development Guide](../runtime/off-chain-workers).
+
+# Off-Chain Indexing
```

Comment:
TBH I'm not sure it's fair to bundle Off-Chain Indexing under Off-Chain Workers. These two things can really be though of independently.

I'd suggest renaming the entire article to "Off-Chain Features" or sth, and then having 3 sections that outline 3 independent substrate features:
- Off-Chain Database - a node-local database that every substrate node has, can be accessed over RPC (both writes (unsafe) and reads (safe)).
- Off-Chain Workers - an idea of running off-chain an arbitrary logic defined within the runtime code (when the node is fully synced). Workers can access Off-Chain DB.
- Off-Chain Indexing - a Runtime function to populate Off-Chain DB (opt-in via CLI flag) even during sync.

Reply:
Thank you both for thinking critically about how we present these concepts to our users :pray: I would not be opposed to Tomek's suggestion at all, and we could even think about "lifting" this out of the Runtime section and creating a new top-level Off-Chain section or something.

Reply:
I will follow tomek suggestion of renaming the section as `Off-Chain Features` and having 3 children sections inside.